config/rbac/role.yaml

```diff
@@ -106,15 +106,6 @@ rules:
   - get
   - list
   - watch
-- apiGroups:
```

Comment:
why are we dropping these? I don't see any corresponding kubebuilder tags being dropped?

Reply:
The entire node controller is being dropped, the RBAC changes are coming from this one.